ci-operator/config/openshift/openstack-cinder-csi-driver-operator/openshift-openstack-cinder-csi-driver-operator-master.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+binary_build_commands: go build cmd/openstack-cinder-csi-driver-operator/main.go
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
```

Comment:
1.15

Reply:
https://github.com/openshift/openstack-cinder-csi-driver-operator/blob/master/Dockerfile#L1

Reply:
then we need to update the dockerfile